pkg/webui/lib/store/actions/init.js

```diff
@@ -18,4 +18,4 @@ export const INITIALIZE_BASE = 'INITIALIZE'
 export const [
   { request: INITIALIZE, success: INITIALIZE_SUCCESS, failure: INITIALIZE_FAILURE },
   { request: initialize, success: initializeSuccess, failure: initializeFailure },
-] = createRequestActions(INITIALIZE_BASE)
+] = createRequestActions(INITIALIZE_BASE, session_id => ({ session_id }))
```

Comment:
This is not required. You don't pass the session ID as part of the request action.